.github/workflows/python.yaml

```diff
@@ -3,7 +3,10 @@ name: Python CI
 on:
   push:
     branches:
-    - master
+      - 'master'
+      - 'release/*'
+    tags:
+      - 'v*'
```

Comment:
Just confirming: only `python.yaml` produces any useful artifacts, which is why it is the only workflow that triggers on tags?

Reply:
Yes. For enterprise, only `packaging.yaml` will run.